app/views/password_reset_mailer/reset_email.text.erb

```diff
@@ -0,0 +1,9 @@
+Hey, <%= @user.username %>!
+===============================================
+ 
+You have successfully requested to reset your password. 
+ 
+To reset your password, please click this link:
+<%= edit_password_reset_url(token: @token) %>.
+
+Thanks for using Haxxor News and have a great day!
```

Comment:
might want a blurb that says "if you didn't request this, then just ignore it...."